docs/gettinginvolved/infrastructure/maintaining-cloudformation.rst

```diff
@@ -34,7 +34,10 @@ Generate CloudFormation JSON template:
 
 .. prompt:: bash #
 
-   admin/create-cloudformation-template > /tmp/flocker-cluster.cloudformation.json
+   admin/create-cloudformation-template \
+       --client-ami-map-body="$(< $PWD/docker.json)" \
```

Comment:
From this command line I sort of expected a `docker.json` and a `flocker.json` file to be checked into ClusterHQ/flocker in the root directory (since `admin/create-cloudformation-template` suggests that's what PWD is).  I didn't find them there though. Is the expectation that these 2 files are created manually by whomever wants to run this script?

That seems really unfortunate. Any chance you want to just check in template files? I'd much rather just update an existing file than write JSON by hand.